app/src/main/java/com/fastaccess/data/dao/CommentRequestModel.java

```diff
@@ -25,7 +27,7 @@ public CommentRequestModel() {}
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         CommentRequestModel that = (CommentRequestModel) o;
-        return position == that.position && (path != null ? path.equals(that.path) : that.path == null);
+        return Objects.equals(position, that.position) && (path != null ? path.equals(that.path) : that.path == null);
```

Comment:
YANKI OBJECTS is JAVA 8 FEATURE & THAT MEANS IT WONT WORK ON ANY ANDROID BUT LATEST!!!!!!!!!